### ISSUE TITLE
a11y violations on documentation site.

### ISSUE THREAD

**Issue by kid-icarus** [emotion-js/emotion#1924]
Can you please take a look at your documentation site and increase the foreground/background contrast on your links and code snippets?

The [axe chrome extension](https://chrome.google.com/webstore/detail/axe-web-accessibility-tes/lhdoppojpmngadmnindnejefpokejbdd) is currently showing multiple WCAG 2 AA contrast ratio threshold violations.

![ezgif com-video-to-gif](https://user-images.githubusercontent.com/864752/86426116-bf150500-bc9b-11ea-95de-b445b3fa69b3.gif)

**Comment by Andarist** [comment]
Thanks for the report - I will take a look at resolving a11y problems.

**Comment by Andarist** [comment]
I've fixed the main contrast ratio violations in https://github.com/emotion-js/emotion/pull/1929 && https://github.com/emotion-js/emotion/pull/1930 . If you find anything more that should still be fixed then let me know.
